modules/chatops4s-examples/src/main/resources/application.conf

```diff
@@ -0,0 +1,17 @@
+slack {
+  # Get these from your Slack app configuration
+  # Environment variables: SLACK_BOT_TOKEN and SLACK_SIGNING_SECRET
+  botToken = "xoxb-your-bot-token-here"
+  botToken = ${?SLACK_BOT_TOKEN}
+
+  signingSecret = "your-signing-secret-here"
```

Comment:
its better to set it to `null` here, thros an error sooner (during config parsing, instead of sending "your-secret..." to the api.

same for bot token